test/integrations/destinations/criteo_audience/dataDelivery/other.ts

```diff
@@ -0,0 +1,257 @@
+    scenario: 'other',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            headers,
+            params,
+            method: 'PATCH',
+            endpoint: 'https://api.criteo.com/2022-10/audiences/34897/contactlist',
+            JSON: {
+              data: {
+                type: 'ContactlistAmendment',
+                attributes: {
+                  operation: 'add',
+                  identifierType: 'madid',
+                  identifiers: ['sample_madid', 'sample_madid_1', 'sample_madid_2'],
+                  internalIdentifiers: false,
+                },
+              },
+            },
+          },
+          [
+            {
+              attemptNum: 1,
+              destinationId: 'dummyDestinationId',
+              dontBatch: false,
+              secret: {},
+              sourceId: 'dummySourceId',
+              userId: 'dummyUserId',
+              workspaceId: 'dummyWorkspaceId',
+              jobId: 1,
+            },
+          ],
+        ),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            status: 500,
+            response: [
+              {
+                error:
+                  '{"errors":[{"traceIdentifier":"80a1a0ba3981b04da847d05700752c77","type":"authorization","code":"audience-invalid"}]}',
+                metadata: {
+                  attemptNum: 1,
+                  destinationId: 'dummyDestinationId',
+                  dontBatch: false,
+                  secret: {},
+                  sourceId: 'dummySourceId',
+                  userId: 'dummyUserId',
+                  workspaceId: 'dummyWorkspaceId',
+                  jobId: 1,
+                },
+                statusCode: 500,
+              },
+            ],
+            message: 'Request Failed: during criteo_audience response transformation (Retryable)',
+            statTags: {
+              destType: 'CRITEO_AUDIENCE',
+              errorCategory: 'network',
+              destinationId: 'dummyDestinationId',
+              workspaceId: 'dummyWorkspaceId',
+              feature: 'dataDelivery',
+              implementation: 'native',
+              errorType: 'retryable',
+              module: 'destination',
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'criteo_audience_other_1',
+    name: 'criteo_audience',
+    description: '[Other]:: Test for checking throttling scenario',
+    successCriteria: 'Should return a 429 status code',
+    scenario: 'other',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            headers,
+            params,
+            method: 'PATCH',
+            endpoint: 'https://api.criteo.com/2022-10/audiences/34898/contactlist',
```

Comment:
If throttling scenario is missing in mock add it then it will be applicable for everyone to test

Reply:
In the common network.ts file